code/game/objects/items/storage/belt.dm

```diff
@@ -256,13 +256,17 @@
 	new /obj/item/storage/bottlecase/bica(src)
 	new /obj/item/reagent_containers/hypospray/advanced/kelotane(src)
 	new /obj/item/storage/bottlecase/kelo(src)
+	new /obj/item/reagent_containers/hypospray/advanced/dylovene(src)
+	new /obj/item/storage/bottlecase/dylo(src)
 	new /obj/item/reagent_containers/hypospray/advanced/tramadol(src)
 	new /obj/item/storage/bottlecase/tram(src)
 	new /obj/item/reagent_containers/hypospray/advanced/tricordrazine(src)
 	new /obj/item/storage/bottlecase/tricord(src)
 	new /obj/item/reagent_containers/hypospray/advanced/dexalin(src)
 	new /obj/item/reagent_containers/glass/bottle/dexalin(src)
 	new /obj/item/reagent_containers/hypospray/advanced/inaprovaline(src)
+	new /obj/item/reagent_containers/hypospray/advanced/spaceacillin(src)
+	new /obj/item/reagent_containers/hypospray/advanced/hypervene(src)
 	new /obj/item/reagent_containers/hypospray/advanced/peridaxon(src)
 	new /obj/item/reagent_containers/hypospray/advanced/quickclot(src)
```

Comment:
all the single hypospray should not be hyposray, but injectors. This should definitly not procure more reagants than normal medbelt